root/layout/components/Footer.js

```diff
@@ -1,95 +1,520 @@
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div className="col-sm-12 col-md-3">
+            <h3 className="fs-3 fw-bold color-black ms-3">
+              {l('Fellow Projects')}
+            </h3>
+            <ul className="list-style-type">
+              <li>
+                <img
+                  alt="ListenBrainz"
+                  className="me-1"
+                  height="24"
+                  src={listenbrainzLogo}
+                  width="24"
+                />
+                <a
+                  className="fw-bold fs-5"
+                  href="https://listenbrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('ListenBrainz')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="CritiqueBrainz"
+                  className="me-1"
+                  height="24"
+                  src={critiquebrainzLogo}
+                  width="24"
+                />
+                <a
+                  className="fw-bold fs-5"
+                  href="https://critiquebrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('CritiqueBrainz')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="Picard"
+                  className="me-1"
+                  height="24"
+                  src={picardLogo}
+                  width="24"
+                />
+                <a
+                  className="fw-bold fs-5"
+                  href="https://picard.musicbrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Picard')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="BookBrainz"
+                  className="me-1"
+                  height="24"
+                  src={bookbrainzLogo}
+                  width="24"
+                />
+                <a
+                  className="fw-bold fs-5"
+                  href="https://bookbrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('BookBrainz')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="AcousticBrainz"
+                  className="me-1"
+                  height="24"
+                  src={acousticbrainzLogo}
+                  width="24"
+                />
+                <a
+                  className="fw-bold fs-5"
+                  href="https://acousticbrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('AcousticBrainz')}
+                </a>
+              </li>
```

Comment:
Given AB is discontinued, I would not link to it.

Reply:
Right